app/views/shared/_practice_cards_row.html.erb

```diff
@@ -0,0 +1,6 @@
+<% classes ||= nil %>
+<div class="grid-row grid-gap-3<%= (' ' + classes) if classes.present? %>">
```

Comment:
⚠️ tried to use this component for the recommended for you practice card list but there were errors so just used this component for page builder pages and partners pages